database/table/tables.go

```diff
@@ -14,8 +14,8 @@ import (
 
 type Workflow struct {
 	gorm.Model
-	Name        string      `gorm:"type:varchar(256);not null"`
-	Artifact    string      `gorm:"type:varchar(2048);not null"`
+	Name        string      `gorm:"type:varchar(256);index:workflows_name_artifact,unique"`
+	Artifact    string      `gorm:"type:varchar(2048);index:workflows_name_artifact,unique"`
```

Comment:
is this creating a combined unique constraint on `(name, artifact)` or `name` and `artifact` separately?

Reply:
create 1 unique constraint on the 2 columns